### PR TITLE
Home Page: Remove current work section and revise contribution section

### PR DESCRIPTION
**Pull request by mcking65**
Several of the resources referenced in the current work section are out-of-date and not ready to be featured on the home page.

This revision is intended to reduce, clarify, and simplify the home page.
___
[WAI Preview Link](https://deploy-preview-95--wai-aria-practices2.netlify.app)

### CODE REVIEW

**Comment by mcking65** [comment]
@isaacdurazo @alflennik 

I am guessing this change might mess up the color scheme a bit so the sections are less visually distinct. Please add appropriate adjustments.

**Comment by mcking65** [comment]
@alflennik 

I don't see a preview link. Under checks I see:

>Trigger PR update for WAI-APG site / create-pr-wai (pull_request_target) Successful in 6s

But, I can't find a PR in wai-aria-practices that corresponds to this PR.

**Comment by richnoah** [comment]
@mcking65 have you merged the PR?

**Comment by mcking65** [comment]
@richnoah, no, the PR is not merged, and there are no signs of a preview for it. That rather constrains reviews.

**Comment by richnoah** [comment]
@mcking65 it turns out there was a build error that was occurring.  Howard has implemented a fix to resolve the build error.  Based on the time left and the body of this work the team is suggesting the use of display:none

**Comment by mcking65** [comment]
@richnoah I have restored the div with id="work" so it can be hidden in downstream processing.

@a11ydoer, @charmarkk, @shirsha, after the display:none is applied to the current work section and a preview link is posted, this PR will be ready for review. Per discussion in today's meeting, that will be later today.

**Comment by richnoah** [comment]
@mcking65 I have been following the commits to make sure the build process is working.  The deploy preview has been updated.  Thank you for making the update.

**Comment by mcking65** [comment]
@a11ydoer, @charmarkk, @shirsha,

The new content is available for review in the [preview](https://deploy-preview-95--wai-aria-practices2.netlify.app/aria/apg/)
under the heading "GET INVOLVED". I revised the information about joining the task force, contributing via GitHub, and the mailing list. Please look for any mistakes or editorial changes that should be made to that copy.

The current work section is not yet hidden (as of 12:55 Pacific). Once it is hidden, it will change the look somewhat. That will happen later today.

File: index.html [hunk 1]
```diff
@@ -155,46 +159,42 @@ <h3>Meetings</h3>
...
         <li>
-          <h3>Contribute to Our Project</h3>
+          <h3>Contribute via GitHub</h3>
           <p>
-            To contribute without joining the task force, see the ARIA Working
-            Group contribute page for general instructions. To contribute to
-            documents under development, see how to contribute to the source
-            repository directly.
+            Many valuable contributions are made by people who find or raise issues of interest in our GitHub repository and then submit proposed changes via a GitHub pull request.
+            If you choose this path, it is tremendously helpful if you first study the APG's contribution guidelines.
```
Comment (a11ydoer) [inline index.html:190]: @mcking65 

The contents look good and the tone is very kind. ;-)
Only one thing is that this sentence can be rewritten. there are two If sentences in one sentence.

**Review by a11ydoer — CHANGES_REQUESTED** [review]
It would be easier to understand if this sentence can be rewritten. 

"If you choose this path, it is tremendously helpful if you first study the APG's contribution guidelines"

**Comment by mcking65** [comment]
@a11ydoer, thank you for the feedback. I rewrote that sentence. Please let me know if the change resolves your concern.

**Comment by shirsha** [comment]
1) Providing elements with accessible names and, where appropriate, accessible descriptions, is one of the most important responsibilities authors have when developing accessible web experiences.

please remove the comma after the accessible description

2) There is an empty <footer> element on the page.
3) The main content on the page is placed in an <header> element which is nested inside the <main> element.
4) Another <header> element is used inside <aside> landmark which contains heading as " Help improve this page ".
5)The aria-label for <footer> element is not descriptive
6)  Some of the content on the home page are missing landmarks.
7) There are more than two lines of centered text. Ex: content under the "Get Involved", "Join the Task Force", and 
"Contribute via Github".
8) The aria-label for all "learn more" links are missing  visible label name.

**Comment by shirsha** [comment]
Decorative icons have alternative text

**Review by a11ydoer — APPROVED** [review]
The change looks good, @mcking65

**Comment by a11ydoer** [comment]
> @a11ydoer, thank you for the feedback. I rewrote that sentence. Please let me know if the change resolves your concern.

**Comment by mcking65** [comment]
@shirsha wrote:

> 1. Providing elements with accessible names and, where appropriate, accessible descriptions, is one of the most important responsibilities authors have when developing accessible web experiences. please remove the comma after the accessible description.

OK

> 2. There is an empty `footer` element on the page.

Please be sure you are looking at the post-processed, rendered DOM in the preview:
https://deploy-preview-95--wai-aria-practices2.netlify.app/aria/apg/

BTW, if the empty footer is there, that would be a template issue, which would not be related to this PR.

> 3. The main content on the page is placed in a `header` element which is nested inside the `main` element.

I don't understand what you mean. I don't see a header wrapping all the main content. There is a header on the H1. Is that what you are talking about? If so, is it a problem? It is part of the WAI site template, and would be out of scope for this PR.

> 4. Another `header` element is used inside the `aside` landmark which contains the heading " Help improve this page ".

That is part of the template also. Is that an error?

> 5. The aria-label for the `footer` element is not descriptive.

Do you mean the label "Site"? Are you thinking it should be "WAI" instead of "Site"? That also is part of the 
WAI site template, and would not be resolved by this PR. I do agree that a better label would be "WAI" or possibly "Web Accessibility Initiative".

> 6. Some of the content on the home page are missing landmarks.

Do you mean the banner content at the top? We have an open issue for that in the template repo.

> 7. There are more than two lines of centered text. Ex: content under the "Get Involved", "Join the Task Force", and
>    "Contribute via Github".

Is this a problem? Please explain.

> 8. The aria-label for all "learn more" links are missing a visible label name.

I don't understand what is missing. You can't provide a visible label with aria-label. Please explain.

> 9. Under Mailing list - The link "view the mailing list archive" in the sentence is not underlined.

Is that inconsistent with other links inside sentences on the page or other places on the site?

> 10. Decorative icons have alternative text.

I think it is reasonable to go either way on the graphics. While I agree that they are not essential to the meaning of the page, they don't feel entirely decorative to me; they convey an asthetic. I wonder if we should get input on this from a variety of screen reader users during our usability study. I am reluctant to remove the names prior to launch.

**Comment by a11ydoer** [comment]
Thanks for your feedback, @shirsha 

I am just adding more info to the responses by @mcking65  I also think Siri is looking at a different view, not the preview of this PR.  Some of Siri's review comments can be added as separate issues or consolidated with the existing ones. 

> > 8. The aria-label for all "learn more" links are missing a visible label name. 
> 
> I don't understand what is missing. You can't provide a visible label with aria-label. Please explain.
Jemma - "Learn more" links looks good since all have good accessible names with aria-label. It does not need a visual label.  
> 
> > 9. Under Mailing list - The link "view the mailing list archive" in the sentence is not underlined.
> 
> Is that inconsistent with other links inside sentences on the page or other places on the site?

Jemma - All the links in the new APG site do not use the underline style.  Therefore, it is consistent throughout the site.

**Comment by shirsha** [comment]
@a11ydoer  @mcking65 
8. The aria-label for all "learn more" links are missing a visible label name.
Matt -I don't understand what is missing. You can't provide a visible label with aria-label. Please explain.
Jemma - "Learn more" links looks good since all have good accessible names with aria-label. It does not need a visual label.
 Siri response- As per WCAG 2.5.3 "label in name" the aria-label must include a visible label along with the string. Ex "learn more about landmark guide"
9. Under Mailing list - The link "view the mailing list archive" in the sentence is not underlined.
Is that inconsistent with other links inside sentences on the page or other places on the site?

Thank Jemma for creating the issue https://github.com/w3c/aria-practices/issues/2327

For easier visibility for low vision and color blind users, we recommend providing a non-color-based cue.

@mcking65 
10. I agree with your comment
 7. Centered text- It decreases readability and affects users with tunnel vision.  There are a lot of usability studies on avoiding centered text.
 5. I will raise a seperate issue for this one
 For #3, #4, #6 - Regarding the <header> element, <footer> element I will check again and raise issue accordingly.




#2327

**Comment by mcking65** [comment]
> @a11ydoer @mcking65 8. The aria-label for all "learn more" links are missing a visible label name. Matt -I don't understand what is missing. You can't provide a visible label with aria-label. Please explain. Jemma - "Learn more" links looks good since all have good accessible names with aria-label. It does not need a visual label. Siri response- As per WCAG 2.5.3 "label in name" the aria-label must include a visible label along with the string. Ex "learn more about landmark guide" 9.

@shirsha thank you for the clarification. Now I understand what you mean. I think you mean to say that it is best practice in this case for the non-visual label to begin with the text of the visible label. I agree with that. Given timing, down to minutes now, we can follow up with that later.

**Comment by mcking65** [comment]
@a11ydoer thank you for opening the related issues for Siri's feedback.

**Pull request merged**